packages/expo-notifications/android/src/main/java/expo/modules/notifications/notifications/model/triggers/FirebaseNotificationTrigger.kt

```diff
@@ -13,7 +13,7 @@ import expo.modules.notifications.notifications.interfaces.NotificationTrigger
 /**
  * A trigger representing an incoming remote Firebase notification.
  */
-class FirebaseNotificationTrigger(val remoteMessage: RemoteMessage) : NotificationTrigger {
+class FirebaseNotificationTrigger(private val remoteMessage: RemoteMessage) : NotificationTrigger {
```

Comment:
It's not needed anymore - see https://github.com/expo/expo/pull/32110.

Reply:
reverted